### DISCUSSION TITLE
Login emails not being sent anymore

### DISCUSSION THREAD

**Discussion by keyranova** [vercel/community#5168] [Account Issue]
### Question

My job has a google workspace set up with a dev email address. (a shared email address that sends mail to multiple dev emails). Logging in used to work fine with this shared email address but at the end of November, vercel stopped sending the emails when attempting to login. Has anyone ran into a similar issue? Is it an issue on Vercel's side of things? I've sent emails to registration@vercel.com but have not received a reply.

**Comment by JafCR** [comment]
I am also trying to login via email and it's not working.
After finishing the sign up process I can't login. It always shows this error msg

![image](https://github.com/vercel/community/assets/33846016/79736f91-f4c8-4aef-a913-84b6c53ce90a)

and [this guide](https://vercel.com/guides/why-can-i-not-signup) is not really helpful
